proto/asgt/type/dataset.proto

```diff
@@ -13,7 +13,8 @@ message Dataset {
   string type = 3;
   repeated string tags = 4;
   repeated string targets = 5;
-  reserved 6 to 8;
+  reserved 6 to 7;
+  google.protobuf.Timestamp truncated_at = 8;
```

Comment:
Be careful with changing the reserved fields... there might be some hidden reason for it... let's just use the latest id which is 13